test/blackbox-tests/test-cases/pkg/pkg-deps.t

```diff
@@ -18,16 +18,12 @@ We should be able to specify (package ..) deps on locally built packages.
   > (dirs :standard \ external_sources)
   > (rule
   >  (alias foo)
-  >  (action (system "command -v foo"))
+  >  (action (which foo))
```

Comment:
Did you mean `run`?

Reply:
Does this get interpreted as a `run`? I wasn't aware `rule` was able to do that.